### PR TITLE
Nodes and clients developer doc: adds note about OpenEthereum being deprecated 

### PR DESCRIPTION
**Pull request by taxmeifyoucan**
Add info on OE being deprecated

### CODE REVIEW

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ On the other hand, if you run a client, you can share it with your friends who m
 
 The Ethereum community maintains multiple open-source clients, developed by different teams using different programming languages. This makes the network stronger and more diverse. The ideal goal is to achieve diversity without any client dominating to reduce any single points of failure.
 
-This table summarizes the different clients. All of them are actively worked on and pass [client tests](https://github.com/ethereum/tests).
+This table summarizes the different clients. All of them pass [client tests](https://github.com/ethereum/tests) and are actively mainntained to stay updated with network upgrades.
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/index.md:110]:
```suggestion
This table summarizes the different clients. All of them pass [client tests](https://github.com/ethereum/tests) and are actively maintained to stay updated with network upgrades.
```

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 2]
```diff
@@ -137,6 +137,8 @@ OpenEthereum is a fast, feature-rich and advanced CLI-based Ethereum client. It'
 
 OpenEthereum is developed using the cutting-edge Rust programming language and licensed under the GPLv3.
 
+**Note that OpenEthereum [has been depracted](https://medium.com/openethereum/gnosis-joins-erigon-formerly-turbo-geth-to-release-next-gen-ethereum-client-c6708dd06dd) and is no longer be maintained.** Use it with caution and preferably switch to another client implementation.
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/index.md:140]:
```suggestion
**Note that OpenEthereum [has been deprecated](https://medium.com/openethereum/gnosis-joins-erigon-formerly-turbo-geth-to-release-next-gen-ethereum-client-c6708dd06dd) and is no longer be maintained.** Use it with caution and preferably switch to another client implementation.
```

**Comment by minimalsm** [comment]
Any reason we shouldn't just rip out OpenEthereum from this page entirely? @taxmeifyoucan

**Comment by taxmeifyoucan** [comment]
@minimalsm That was my initial idea but OE still works properly, is used a lot by community and services. Importantly, it supports current network changes and even [upcoming Glacier](https://blog.ethereum.org/2021/11/10/arrow-glacier-announcement/). Glacier blog mentions it with warning, I would keep it that way here and remove completely in next months, before the Merge.

**Comment by minimalsm** [comment]
You got it! Thanks again @taxmeifyoucan

**Pull request merged**